dist/index.html

```diff
@@ -5,7 +5,23 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/assets/interceptor.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Home | The Interceptor</title>
+  <title>The Interceptor | Dark Florist</title>
+  <link rel="canonical" href="https://www.dark.florist">
```

Comment:
```suggestion
  <link rel="canonical" href="https://dark.florist">
```

Reply:
Is a centralized server required for this, it cannot work with just a relative path or anything?